app/src/main/res/layout/dashboard_contract_row.xml

```diff
@@ -25,7 +25,6 @@
         android:drawablePadding="@dimen/base_margin"
         android:gravity="center_vertical"
         android:textAppearance="?textAppearanceBody2"
-        tools:drawableStart="@drawable/ic_active"
         tools:text="Aktiv" />
 
     <com.hedvig.app.ui.view.HedvigCardView
```

Comment:
We're actually back on `MaterialCardView`s now - you can change all uses of `HedvigCardView` on this screen to make it look correct 🙂